src/fundus/publishers/de/junge_welt.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+
+class JungeWeltParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div > p")
+        _summary_selector = CSSSelector(".teaser")
+        _subheadline_selector = XPath("//div[@class = 'kicker']/div[2]")
+
+        @attribute
+        def body(self) -> ArticleBody:
+            return extract_article_body_with_selector(
+                self.precomputed.doc,
+                summary_selector=self._summary_selector,
+                subheadline_selector=self._subheadline_selector,
+                paragraph_selector=self._paragraph_selector,
+            )
+
+        @attribute
+        def authors(self) -> List[str]:
+            return generic_author_parsing(self.precomputed.ld.bf_search("author"))
```

Comment:
I would suggest using the meta tag "Author" in this case, because the JSON seems to be incomplete or missing in some cases and the meta seems more reliable :)

Reply:
Thanks for the suggestion, i have fixed it.